### DISCUSSION TITLE
最近有人在ws+tls+web的模式下被封443吗?(讨论下是否和xray的vless回落有关)

### DISCUSSION THREAD

**Discussion by HiColinn** [v2fly/v2ray-core#1115] [General]
想给大家讨论个问题,不知道你们最近遇到过没有;
我用v2ray的ws+tls+web的方式,用了将近两年,最严的时候也从来没被封过.大概两个月之前,我迁移到了xray,因为看着vless的回落功能很强大,即xray监听443,根据域名处理不同的流量,分流等,然后回落给nginx,但是大概一周前,突然不能用了,我从来没怀疑过443端口被封.
然后又重回v2ray,还是用以前的ws+tls+web的方式,搞了两天两夜都不知道怎么回事,刚才再v2ray的讨论板块,有个好心人告诉我可能是443被封了,她说最近很多人遇到了。
所以我想问问大家最近有没有被封443？是否跟xray的vless回落有关？因为我发现xray已经几个月没更新了。

**Comment by 33lyl** [comment]
暂时没发现
挂一层cdn吧

**Reply by qiangliu8183** [reply]
和你情况一样，怀疑是xtls的问题，生日那两天所有xray的小鸡都被掐死了，v2ray的正常，和共享不共享没关系，和流量大小也没关系

**Comment by aigooman** [comment]
求问过段时间会解封么，考虑要不要换ip……
